src/main/java/appeng/core/Registration.java

```diff
@@ -416,20 +336,15 @@ public void registerRecipes( RegistryEvent.Register<IRecipe> event )
 
 		final Api api = Api.INSTANCE;
 		final ApiDefinitions definitions = api.definitions();
+		final Side side = FMLCommonHandler.instance().getEffectiveSide();
 
 		// Perform ore camouflage!
 		ItemMaterial.instance.makeUnique();
 
-		final Runnable recipeLoader = new RecipeLoader( this.recipeDirectory, this.customRecipeConfig, this.recipeHandler );
-		recipeLoader.run();
-
-		this.recipeHandler.injectRecipes();
-
 		if( AEConfig.instance().isFeatureEnabled( AEFeature.ENABLE_DISASSEMBLY_CRAFTING ) )
 		{
 			DisassembleRecipe r = new DisassembleRecipe();
-			// TODO : 1.12 Improve
-			addRecipeToRegister( r.setRegistryName( AppEng.MOD_ID.toLowerCase(), "disassemble" ) );
+			registry.register( r.setRegistryName( AppEng.MOD_ID.toLowerCase(), "disassemble" ) );
 			// RecipeSorter.register( "appliedenergistics2:disassemble", DisassembleRecipe.class, Category.SHAPELESS,
 			// "after:minecraft:shapeless" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)